dist/rockcraft-pack-action/index.js

```diff
@@ -4219,20 +4219,20 @@ async function ensureLXD() {
     await exec.exec('sudo', ['lxd', 'init', '--auto']);
     await ensureLXDNetwork();
 }
-async function ensureRockcraft(channel) {
+async function ensureRockcraft(channel, revision) {
     const haveRockcraft = await haveExecutable('/snap/bin/rockcraft');
     core.info('Installing Rockcraft...');
     await exec.exec('sudo', [
         'snap',
         haveRockcraft ? 'refresh' : 'install',
-        '--channel',
-        channel,
+        revision.length > 0 ? '--revision' : '--channel',
```

Comment:
this means that revision will silently mask channel, can we make it a bit more obvious that it is ignored or make it mutually exclusive in whatever schema we have?

A log message should be enough to start out.

Reply:
y good idea. I've added a log message in d9c1181. Making them mutually exclusive could be weird cause `channel` is not a user-defined property, but rather a fallback.